templates/travel/partials/activities/activities.snip.html

```diff
@@ -0,0 +1,54 @@
+{{!
+<!---
+Copyright 2016 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+<!-- Activities -->
+<section class="travel-activities pb4 pt3 relative">
+  <div class="max-width-3 mx-auto px1 md-px2">
+    <h3 class="bold h1 line-height-2 mb2 md-hide lg-hide">Browse<br>by activity</h3>
+    <h3 class="bold h1 line-height-2 mb2 xs-hide sm-hide center">Browse by activity</h3>
```

Comment:
can you make one of this aria-hidden , so that screen readers don't read it twice (i would prefer the first one)

Reply:
Done